extensions/amp-bind/0.1/amp-state.js

```diff
@@ -165,14 +165,11 @@ export class AmpState extends AMP.BaseElement {
    * @private
    */
   fetch_(ampdoc, policy, opt_refresh, token = undefined) {
-    return batchFetchJsonFor(
-      ampdoc,
-      this.element,
-      /* opt_expr */ undefined,
-      policy,
-      opt_refresh,
-      token
-    );
+    return batchFetchJsonFor(ampdoc, this.element, {
+      urlReplacement: policy,
+      refresh: opt_refresh,
+      token: token,
```

Comment:
`token`, the linter should warn about this.

Reply:
done.